src/Goerli-DssSpell.sol

```diff
@@ -23,27 +23,120 @@ import "dss-exec-lib/DssAction.sol";
+        DssExecLib.setIlkStabilityFee("WSTETH-A", TWO_TWO_FIVE_PCT_RATE, true);
+
+        // Decrease the CRVV1ETHSTETH-A Stability Fee from 3.5% to 2.25%
+        // NOTE: ignore in goerli
+        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", TWO_TWO_FIVE_PCT_RATE, true);
+
+        // Decrease the WBTC-A Stability Fee from 3.75% to 3.25%
+        DssExecLib.setIlkStabilityFee("WBTC-A", THREE_TWO_FIVE_PCT_RATE, true);
+
+        // Decrease the WBTC-B Stability Fee from 5.0% to 4.5%
+        DssExecLib.setIlkStabilityFee("WBTC-B", FOUR_FIVE_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC1-A Stability Fee from 0.1% to 0%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC2-A Stability Fee from 0.25% to 0.05%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC2-A", ZERO_ZERO_FIVE_PCT_RATE, true);
+
+
+        // ---------------------- Debt Ceiling Updates -------------------------
+
+        // https://forum.makerdao.com/t/immediate-short-term-parameter-changes-proposal-for-crvv1ethsteth-a-dc-and-gap-increase/14476 
+        // Increase the CRVV1ETHSTETH-A Maximum Debt Ceiling from 3 million DAI to 5 million DAI.
+        // NOTE: ignore in goerli
+        // DssExecLib.setIlkAutoLineDebtCeiling("CRVV1ETHSTETH-A", 5 * MILLION);
+
+        // https://vote.makerdao.com/polling/QmdS8mCx#poll-detail
+        // Increase the GUNIV3DAIUSDC1-A Maximum Debt Ceiling from 100 million DAI to 750 million DAI.
+        // Increase the GUNIV3DAIUSDC1-A gap from 10 million to 50 million
+        // Leave the GUNIV3DAIUSDC1-A ttl the same
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 750 * MILLION, 50 * MILLION, 8 hours);
+
+        // https://vote.makerdao.com/polling/QmdS8mCx#poll-detail
+        // Increase the GUNIV3DAIUSDC2-A Maximum Debt Ceiling from 750 million DAI to 1 billion DAI.
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC2-A", 1 * BILLION);
+
+
+        // ----------------------- Target Borrow Rates -------------------------
+        // https://vote.makerdao.com/polling/QmdS8mCx#poll-detail 
+
+        // Increase the DIRECT-AAVEV2-DAI target borrow rate from 2.85% to 3.5%
+        // NOTE: ignore in goerli
+        // DssExecLib.setD3MTargetInterestRate(DssExecLib.getChangelogAddress("MCD_JOIN_DIRECT_AAVEV2_DAI"), 350); // 3.5%
+
+
+        // ------------------------ OSM auth updates ---------------------------
+        // Adds two oracle wallets to all the OSMs
+        // https://discord.com/channels/893112320329396265/897479589171986434/960938185800683580
+        bytes32[] memory ilks = RegistryLike(DssExecLib.reg()).list();
+        for(uint256 i = 0; i < ilks.length; i++) {
+            uint256 class = RegistryLike(DssExecLib.reg()).class(ilks[i]);
+            if (class != 1) { continue; }
+
+            address pip = RegistryLike(DssExecLib.reg()).pip(ilks[i]);
+            // skip USDC, TUSD, PAXUSD, GUSD
```

Comment:
What's the reason for skipping these?

If we're already giving them auth status on the OSM's what's the (additional) harm they can do by having access to these?